sdk-core/transactions-outcome-parsers/interfaces.md

```diff
@@ -0,0 +1,33 @@
+## Interfaces needed for parsing token management transactions
+
+```
+dto ITransactionEvent:
+    address: string;
+    identifier: string;
+    topics: List[string]
+    data: string
+```
+
+```
+dto ITransactionLogs:
+    address: string;
+    events: List[ITransactionEvent];
+```
+
+```
+dto ITransactionResult:
+    hash: string;
+    timestamp: int;
+    sender: string;
+    receiver: string;
+    data: string;
+    original_tx_hash: string;
+    minblock_hash: string;
```

Comment:
Hopefully, we do no need the miniblock hash - maybe remove it, until we learn is necessary? Also, the `original_tx_hash`, for now?

Reply:
Removed `miniblock_hash` and `original_tx_hash`.